docs/references/sdk/frontend-only.mdx

```diff
@@ -1,7 +1,152 @@
 ---
-description: TODO
+description: A reference for implementing a frontend-only Clerk SDK
 ---
 
 # Frontend-only SDK
 
-TODO
+While [ClerkJS](/docs/references/javascript/overview) can be used in any browser context and framework, realistically users expect to consume its features through the conventions and syntax of their framework of choice. `@clerk/clerk-react` turns ClerkJS into React components, `@clerk/astro` into Astro components, and so on.
+
+In non-browser environments you’ll need to re-implement the [Clerk class](/docs/references/javascript/clerk/clerk) in the SDK’s programming language, interacting with the [FAPI](/docs/references/sdk/terminology).
+
+## Expected features
+
+- User only needs to provide their [publishable key](/docs/references/sdk/terminology)
+- User only needs to adjust one or two files to add Clerk to their app (e.g. adding Clerk to the configuration file of that framework)
+- User can use [Clerk’s components](/docs/components/overview) in their choice of framework (e.g. in a React-based framework you import these components as React components)
+- Give users access to `client`, `session`, `user`, and `organization` properties through the framework’s choice of state management
```

Comment:
Should we link each resource to their respective pages? (i.e. [`session`](https://clerk.com/docs/references/javascript/session), [`user`](https://clerk.com/docs/references/javascript/user/user))